### ISSUE TITLE
Galois group server errors on beta

### ISSUE THREAD

**Issue by AndrewVSutherland** [LMFDB/lmfdb#3028]
http://beta.lmfdb.org/GaloisGroup/46T5 throws a server error (not a problem on production because 46T5 is not in the production database).

I encountered this by clicking the random Galois group button.

There are also some Galois groups whose pages take so long to load that they timeout on beta, for an example try http://beta.lmfdb.org/GaloisGroup/16T1949.

**Comment by jwj61** [comment]
I have removed transitive groups from the database where the degree > 23.  They can easily be reloaded in the future.  So, for degree 46, the problem is resolved

Some would take unreasonably long to load because there are so many other representations -- I saw one with over 180000 of them.  So, we will have to adjust how we display that before reloading those groups.

For groups which are not in the database, we should produce a more useful error message (it currently just says "Error").

For the degree 16 group, it is because we compute the character table on the fly.  We have a limit on the order of the group for this.  I lowered the limit in pull request #3029 .  At the meeting later this month, perhaps we will precompute and store the character tables which would be a better solution.
